manifest_base.yml

```diff
@@ -14,4 +14,4 @@ applications:
     host: eregsnc-demo
   - name: eregs-worker
     no-route: true
-    command: celery worker --app notice_and_comment
+    command: celery worker --app notice_and_comment -l DEBUG
```

Comment:
We should be all right (for now!), since the child manifests don't touch the `applications` key. If we want different log levels in different environments, then we start having to copy and paste.